doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -47,6 +47,8 @@ This error could be caused by the following issues:
 
   * An out-of-tree external flash driver is not selecting :kconfig:option:`CONFIG_PM_EXTERNAL_FLASH_HAS_DRIVER`, resulting in partitions for images located in the external flash memory being not accessible.
 
+  * Insufficient value set for ``CONFIG_BOOT_MAX_IMG_SECTORS``, as MCUboot typically increases slot sizes when external flash is enabled.
```

Comment:
"MCUboot typically increases slot sizes when external flash is enabled" -- This information is not mentioned anywhere else but here. Shouldn't this be mentioned in the main part of the doc, not in troubleshooting?